folly/detail/base64_detail/Base64Api.cpp

```diff
@@ -31,10 +31,16 @@ Base64RuntimeImpl base64EncodeSelectImplementation() {
         base64URLDecodeSWAR};
   }
 #endif
+#if defined(__POWERPC__) // PowerPC BE
```

Comment:
Does this get defined for PPC64 LE, or only 32-bit PPC BE?

Reply:
Also would prefer to check for FOLLY_PPC if this is only ppc 32-bit be.

Reply:
@Orvid `__POWERPC__` includes `__ppc__` and `__ppc64__` – only Big endian, and for practical purposes only Darwin. (I think it was also used on BeOS.)

I generally try to be specific, since I have no way to test on *BSD, AIX or Linux, which could also use `ppc32`. Perhaps `FOLLY_PPC_BE` may do, since we want to have macOS in one define, but do not include `ppc64le` (well, not in general case).

Reply:
Added a define for non-Apple PPC Big endian. Hope it is correct.